test/test_functionalization.py

```diff
@@ -1448,6 +1448,7 @@ def forward(self, arg0_1, arg1_1, arg2_1):
     return view_5
     """)  # noqa: B950
 
+
```

Comment:
You also probably want is_conj / is_neg unit tests. I'm convinced they work, but not bad to test them so it's easy for someone who breaks the code in the future to figure out.